src/ui/Facet/FacetSettings.ts

```diff
@@ -327,10 +330,11 @@ export class FacetSettings extends FacetSort {
     directionAscendingSection.appendChild(iconAscending);
     directionAscendingSection.appendChild(directionItemsAscending);
     new AccessibleButton()
-      .withElement(directionAscendingSection)
-      .withLabel(l('Ascending'))
-      .withSelectAction((e: Event) => this.handleDirectionClick(e, 'ascending'))
+      .withElement(ascending)
+      .withoutLabel()
```

Comment:
These labels were redundant since those elements already contained the same text.